### ISSUE TITLE
brew uninstall does not fix bashrc and bash_profile

### ISSUE THREAD

**Issue by smarts** [withfig/fig#1857]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and tested again (tell us if that fixed it)

### Description:

Running `brew uninstall fig` resulted in no errors, but failed to remove the lines that the installation added to `~/.bashrc` and `~/.bash_profile`, which means that running `source {either of those files}` fails.

Disclaimer: Obviously i did not run `fig doctor` or `fig restart` because this problem occurs **after** uninstallation.

### Environment

```yaml
OS: macOS Monterey
OS version: 12.6
Processor: Intel chip
Info from homebrew's site at time of install/uninstall:
[/api/cask/fig.json (JSON API)](https://formulae.brew.sh/api/cask/fig.json)
[Cask code](https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/fig.rb) on GitHub
Current version: [1.0.61,528](https://versions.withfig.com/fig%20528.dmg)
Requires: macOS >= 10.13
```
